util/storage-samples/storage-owner-sample/src/main/java/org/fidoalliance/fdo/storage/OwnerDbStorage.java

```diff
@@ -376,28 +377,8 @@ public int getMaxOwnerServiceInfoMtuSz() {
       throw new RuntimeException(e);
     }
 
-    sql = "SELECT OWNER_SERVICE_INFO_MTU_SIZE FROM TO2_DEVICES WHERE GUID = ?;";
-
-    try (Connection conn = dataSource.getConnection();
-        PreparedStatement pstmt = conn.prepareStatement(sql)) {
-
-      pstmt.setString(1, getGuid().toString());
-      try (ResultSet rs = pstmt.executeQuery()) {
-        if (rs.next()) {
-          ownerServiceInfoMtuSize = rs.getInt(1);
-        }
-      }
-    } catch (SQLException e) {
-      throw new RuntimeException(e);
-    }
-
-    if (ownerMtuThreshold < ownerServiceInfoMtuSize) {
-      System.out.println(
-          "Device Maximum MTU request exceeds, owner threshold MTU size. "
-              + "MTU size set to owner threshold MTU size");
-    }
-
-    return Math.min(ownerServiceInfoMtuSize, ownerMtuThreshold);
+    // Restricting minimum threshold to 1300 bytes
+    return Math.max(Const.DEFAULT_SERVICE_INFO_MTU_SIZE, ownerMtuThreshold);
```

Comment:
Is there a check on the maximum threshold value?

Reply:
Since the threshold value is accepted through database, database takes care of checking the maximum threshold value before updating the entry.